include/inexor/vulkan-renderer/wrapper/window.hpp

```diff
@@ -10,9 +10,14 @@ namespace inexor::vulkan_renderer::wrapper {
 
 /// @brief RAII wrapper class for GLFW windows.
 class Window {
+public:
+    enum class WindowMode { eWindowed, eFullscreen, eWindowedFullscreen };
```

Comment:
```suggestion
    enum class Mode { WINDOWED, FULLSCREEN, WINDOWED_FULLSCREEN};
```
Note: `Window::WindowMode` only one window needed ;)

Reply:
I originally named it Mode, just wanted to be more specific :) This is good too!